scripts/fwsize.py

```diff
@@ -38,13 +38,25 @@ def process_elf(self):
         return 0
 
     def process_bin(self):
+        BT_STACK_START = int("0x080D7000", 16)
+        BT_STACK_NAME = "stm32wb5x_BLE_Stack_light_fw"
```

Comment:
use variables from fbt environment? And pass them as args
COPRO_STACK_BIN, COPRO_STACK_ADDR, .bin parser